GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -512,7 +513,12 @@ private void ResetSelectedItemsTo(bool resetToParent, bool resetAndDelete)
 
                     IReadOnlyList<GitItemStatus> resetItems = (resetToParent ? selectedItems.Items()
                         : selectedItems.Items().Select(item => item.InvertStatus())).ToList();
-                    Module.ResetChanges(id, resetItems, resetAndDelete: resetAndDelete, _fullPathResolver, filesInUse: out _, output: out _);
+                    Module.ResetChanges(id, resetItems, resetAndDelete: resetAndDelete, _fullPathResolver, out StringBuilder output);
+
+                    if (!string.IsNullOrEmpty(output.ToString()))
+                    {
+                        MessageBox.Show(this, output.ToString(), TranslatedStrings.ResetChangesCaption, MessageBoxButtons.OK, MessageBoxIcon.Error);
```

Comment:
What does `output.ToString()` contain? Is it raw git output? 
Do we need to add some "user friendly" preamble?

Reply:
The text is formatted in ResetChanges() and is partly user friendly. Consider it as a popup that something is wrong. If you have not been bothered with the popup in FormCommit, you will not be bothered by this.